extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -361,6 +378,29 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     });
   }
 
+  /**
+   * Handles Fluid-related messages dispatched from SafeFrame.
+   * @param {!JsonObject} data
+   * @private
+   */
+  receiveMessageForFluid_(data) {
+    const payload = tryParseJson(data['p']);
+    if (!payload || !payload['height']) {
```

Comment:
Nit: `parseInt` to ensure it's not a string `"50px"`.

Reply:
Done.